Sources/InstrumentationBaggage/Baggage.swift

```diff
@@ -13,6 +13,12 @@
 //
 //===----------------------------------------------------------------------===//
 
+#if swift(>=5.5) && canImport(_Concurrency)
+public typealias Baggage_Sendable = Swift.Sendable
+#else 
+public typealias Baggage_Sendable = Any
+#endif
```

Comment:
using this "shim" trick

Reply:
should this be _BaggageSendable to mark it as pseudo-private

Reply:
Will do 👍